examples/ensemble/plot_gradient_boosting_categorical.py

```diff
@@ -156,11 +172,15 @@ def plot_results(figure_title):
             ordinal_result[key],
             native_result[key],
         ]
+
+        mape = [np.mean(np.abs(item)) for item in items]
+        std_pred = [np.std(item) for item in items]
```

Comment:
Not sure what `std_pred` stands for. Maybe the following would be more explicit:

```suggestion
        mape_cv_mean = [np.mean(-item) for item in items]
        mape_cv_std = [np.std(item) for item in items]
```

the lines below will also need to be adjusted.